client/src/galapagotchi.tsx

```diff
@@ -84,16 +87,19 @@ export class Galapagotchi extends React.Component<IAppProps, IGalapagotchiState>
         this.subs.push(this.stateSubject.subscribe(appState => {
             const homeHexalot = appState.homeHexalot
             if (homeHexalot) {
-                location.replace(`/#/${homeHexalot.id}`)
                 const spotCenters = homeHexalot.spots.map(spot => spot.center)
                 const surface = homeHexalot.spots.map(spot => spot.surface === Surface.Land)
                 this.fabricKernel.setHexalot(spotCenters, surface)
-            } else {
-                location.replace("/#/")
             }
             this.setState({appState})
         }))
-        this.fetchIsland("rotterdam")
+        this.props.storage.getOwnedLots().then(ownedLots => {
```

Comment:
async/await?

Reply:
this is within `componentWillUnmount?(): void;` which i doubt i can make async

Reply:
Why not? Nothing much changes from React's perspective

Reply:
The only issue is Typescript might not like it, in which case yeah don't bother

Reply:
My point.  The signature is sync.